examples/pipelines/guestbook.yaml

```diff
@@ -8,76 +8,99 @@ spec:
         - name: build-guestbook                  # 1.a Build guestbook go sample code.
           taskRef:
               name: build-push
-          sourceBindings:
-              - inputName: workspace
+          inputSourceBindings:
+              - name: workspace
                 sourceKey: guestbook
-          artifactStoreBindings:
-              - storeName: registry
-                storeKey: stagingRegistry
-                builtImage: gb-frontend          # TODO Add Commit SHA
+          outputSourceBindings:
+              - name: gb-frontend
+                sourceKey: stagingRegistry
           params:
               - name: pathToDockerfile
-                value: guestbook-go/Dockerfile
+                type: guestbook-go/Dockerfile
         - name: build-redis                      # 1.b Build and push redis docker image.
           taskRef:
               name: build-push
-          sourceBindings:
-              - inputName: workspace
+          inputSourceBindings:
+              - name: workspace
                 sourceKey: redis-docker          # TODO Add Commit SHA
-          artifactStoreBindings:
-              - storeName: registry
-                storeKey: stagingRegistry
-                builtImage: redis4
+          outputSourceBindings:
+              - name: redis
+                sourceKey: stagingRegistry
           params:
               - name: pathToDockerfile
-                value: 4/debian9/4.0/Dockerfile
+                type: 4/debian9/4.0/Dockerfile
         - name: deploy-bundle-test               # 2. Deploy GuestBook and Redis to test cluster
           taskRef:
               name: deploy-with-kubectl
-          sourceBindings:
-              - inputName: workspace
-                sourceKey: guestbook
+          inputSourceBindings:
+            - name: workspace
+              sourceKey: guestbook
+              passedConstraint: 
+                - build-guestbook
+                - build-redis
+            - name: workspace
+              sourceKey: redis-docker   
+              passedConstraint:
+               - build-push
           params:
               - name: pathToFiles
-                value: guestbook/all-in-one/guestbook-all-in-one.yaml
+                type: guestbook/all-in-one/guestbook-all-in-one.yaml
```

Comment:
this is probably by mistake?

Reply:
this field was updated to type in PR #54, we used it to make the tests pass until we get a chance to ask about it

Reply:
As far as I can tell, this should still be `value`, see [`PipelineTaskParam`](https://github.com/knative/build-pipeline/blob/801db1da0f0a483c29c9e9d30dea813281c05f0f/pkg/apis/pipeline/v1beta1/pipeline_types.go#L71).

Reply:
The tests were failing when it was ‘value’